security/pkg/pki/ra/k8s_ra_test.go

```diff
@@ -57,9 +57,9 @@ e+5z6MTAO6ktvHdQlSuH6ARn47bJrZOlkttAhg==
 )
 
 var (
-	testCsrHostName       string = spiffe.Identity{TrustDomain: "cluster.local", Namespace: "default", ServiceAccount: "bookinfo-productpage"}.String()
-	TestCACertFile        string = "../testdata/example-ca-cert.pem"
-	mismatchCertChainFile string = "../testdata/cert-chain.pem"
+	testCsrHostName       = spiffe.Identity{TrustDomain: "cluster.local", Namespace: "default", ServiceAccount: "bookinfo-productpage"}.String()
+	TestCACertFile        = "../testdata/example-ca-cert.pem"
```

Comment:
var-declaration: should omit type string from declaration of var TestCACertFile; it will be inferred from the right-hand side